src/main/php/facilities.php

```diff
@@ -0,0 +1,102 @@
+<?php
+
+include("constants/constants.php");
+
+?>
+
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Our Gym | Facilities</title>
+    <link rel="stylesheet" href="../resources/css/style.css">
+    <link rel="icon" type="image/x-icon" href="../resources/media/dumbbell-logo-black.ico">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,500;0,700;1,100;1,200;1,300;1,600&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+<body>
+
+<section class="section-facilities">
+<?php include ("universal-header.php"); ?>
+
+    <div class="row">
+
+        <div class="column">
+            <img class="image__img" src="../resources/media/cardio.jpg" alt="cardio-room">
+            <div class="image__overlay">
+                <div class="image__title">Cardio</div>
+                <p class="image__description">A modern, well equipped cardio suite!</p>
+            </div>
+        </div>
+
+        <div class="column">
+            <img class="image__img" src="../resources/media/swimming.jpg" alt="swimming-pool">
+            <div class="image__overlay">
```

Comment:
do you need two "_" in the class names?

Reply:
CSS classes updated in my most recent push. Also removed redundant classes in the css